llama_stack/cli/stack/run.py

```diff
@@ -73,10 +73,23 @@ def _add_arguments(self):
             "--image-type",
             type=str,
             help="Image Type used during the build. This can be either conda or container or venv.",
-            choices=["conda", "container", "venv"],
-            default="conda",
+            choices=[e.value for e in ImageType],
         )
 
+    # If neither image type nor image name is provided, but at the same time
```

Comment:
Should we log this behavior to notify users that a decision was made on their behalf?